### PR TITLE
corrected code

### PR DESCRIPTION
**Pull request by BadMikkY**
corrected code&created DB

### CODE REVIEW

File: .idea/vcs.xml
Comment (MaxSkr-iTechArt) [inline .idea/vcs.xml:1]: add to gitignore

File: app/src/main/java/com/example/news_pract/MainActivity.kt [hunk 1]
```diff
@@ -2,10 +2,17 @@ package com.example.news_pract
...
+import com.example.news_pract.database.UsersDao
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var database: AppDataBase
+    private lateinit var dao: UsersDao
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-    }
+        database = getDatabase(this)
```
Comment (MaxSkr-iTechArt) [inline app/src/main/java/com/example/news_pract/MainActivity.kt]: Remove the code that was introduced for testing purposes

File: app/src/main/java/com/example/news_pract/data/RetrofitInstance.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.example.news_pract.data
+
+class RetrofitInstance {
+    companion object {
+        private val retrofit by lazy {
```
Comment (MaxSkr-iTechArt) [inline app/src/main/java/com/example/news_pract/data/RetrofitInstance.kt]: what is this empty delegated property for?

File: app/src/main/java/com/example/news_pract/database/DataBaseHelper.kt [hunk 3]
```diff
@@ -4,7 +4,7 @@ import android.content.Context
 import androidx.room.Room
 
 
-object  DataBaseHelper {
+object DataBaseHelper {
```
Comment (MaxSkr-iTechArt) [inline app/src/main/java/com/example/news_pract/database/DataBaseHelper.kt:7]: you can remove this and use AppDataBase instead

File: app/src/main/java/com/example/news_pract/util/Constants.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package com.example.news_pract.util
+
+class Constants {
+    companion object{
+        const val API_KEY = "b59009fe30ee41aeb0cc57f1e5f309f8"
```
Comment (MaxSkr-iTechArt) [inline app/src/main/java/com/example/news_pract/util/Constants.kt:5]: you can make it a global constant. You don't need the companion object and the Constants class

**Review by VladislavKhilko — APPROVED** [review]

**Pull request merged**